src/app/oauth2/middleware.js

```diff
@@ -42,8 +42,7 @@ module.exports = {
             headers: headers,
           }
         );
-
-        req.session.tokenId = apiResponse?.data["session-id"];
+        req.session.tokenId = apiResponse?.data["session_id"];
```

Comment:
bug fix - api response is returning session_id rather than session-id. 

Action: Should we should check consistence across all BE/FE apis?

Reply:
we should figure out what style we want to use in another PR